### PR TITLE
RT-5.7 TestPlan: Aggregate All Not Viable - README for new test

### PR DESCRIPTION
**Pull request by rszarecki**
README for test that verify behavior when all LAG memeber as set as forwarding-viable = FALSE

### CODE REVIEW

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    - Ensure there is no traffic transmitted form DUT ports 3-7
+    - ensure that traffic is received on all port2-7 and delivered to ATE/OTG port1
+    - ensure there are no packet losses in steady stATE/OTG (no congestion).
+    - Ensure there is no traffic received on DUT LAG_3
+    - Ensure there is no traffic transmitted on DUT LAG_3
+  - Disable/deactivATE/OTG laser on ATE/OTG port2; All LAG members are either down or
+    set with forwarding-viable=FALSE
+    - Ensure ISIS adjacency is UP on DUT LAG_2 and ATE/OTG LAG_2
+    - Ensure there is no traffic transmitted form DUT ports 2-7
+    - ensure that traffic is received on all port2-7 and delivered to ATE/OTG LAG_1
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:67]: But you also said that the laser is off?

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# RT-5.7 Aggregate Not Viable All
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:1]: I think the summary should be something like: "Test forwarding-viable with LAG and routing"?

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+|            |    : ;     |             |       | |        |    (  pfx4   )
+|            |     '      |             |       | |        |     `-------'|
+|            |  LAG_1     |             +-------+-+--------+              |
++------------+            +-------------+ p9    : ;   p9   +--------------+
+                                                 '
+                                                LAG_3
+
+```
+
+- Connect ATE/ATE port-1 to DUT port-1, and ATE/ATE ports 2 through 7 to DUT ports 2-7,
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:38]: replace `ATE/ATE` with `ATE`

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    - ensure there are no packet losses in steady state (no congestion).
+    - Ensure there is no traffic received on DUT LAG_3
+    - Ensure there is no traffic transmitted on DUT LAG_3
+  - Disable/deactive laser on ATE port2; All LAG_2 members are either down (port2) or
+    set with forwarding-viable=FALSE
+    - Ensure ISIS adjacency is UP on DUT LAG_2 and ATE LAG_2
+    - Ensure there is no traffic transmitted out of  DUT ports 2-7 (LAG_2)
+    - ensure that traffic is received on all port3-7 and delivered to ATE/ATE LAG_1
+    - ensure there are no packet losses in steady state (no congestion) for
+      traffic prom ATE LAG_2 to ATE LAG_1 (pfx_1).
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:68]: `prom` -> `from`

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    - Ensure that traffic from ATE port1 to pfx2, pfx3 are transmitted via DUT
+      LAG3
+    - Ensure that traffic from ATE port1 to pfx4 are discarded on DUT
+  - Make the forwarding-viable transitions from FALSE --> TRUE on a ports 7
+    within the LAG_2 on the DUT
+    - ensure that only DUT port 7 of LAG ports has bidirectional traffic.
+    - Ensure there is no traffic transmitted out of  DUT ports 2-6
+    - ensure that traffic is received on all port3-7 and delivered to ATE/ATE port1
+    - ensure there are no packet losses in steady state (no congestion).
+    - Ensure there is no traffic received on DUT LAG_3
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:81]: same, why no traffic on `LAG_3`?

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    - Ensure there is no traffic transmitted on DUT LAG_3
+  - Enable/activATE/ATE laser on ATE port2; Make the forwarding-viable transitions
+    from FALSE --> TRUE on a ports 3-7
+
+### Deviation option
+
+It is foreseen that implementation may drop ISIS adjacency after some members of
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:88]: please clarify the wording.

**Review by xw-g — COMMENTED** [review]
need an issue to track the implementation.

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md (hunk 6 above)
Comment (rszarecki) [reply feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md]: done

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md (hunk 5 above)
Comment (rszarecki) [reply feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:82]: I make separate test procedure for cahe when cost of LAG_2 < LAG_3 and separate when costs are equall.

File: feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
+# RT-5.7 Aggregate Not Viable All
+
+## Summary
```
Comment (xw-g) [inline feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md:3]: @rszarecki can you link your issue here (as a TODO) here? (the one that's tracking test implementation?) This is for the tracking purpose.

**Review by xw-g — APPROVED** [review]

**Pull request merged**